### PR TITLE
🎨 Create the consultancy hero section #176

### LINKED ISSUE #176: 🎨 Create the consultancy hero section
#  Create the consultancy hero section 
---

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user, I want a nice hero section on the consultancy page that captures my attention and introduces the page content. Reuse the text-content component created in #20, and the reusable slider button in  #24. I want a good experience on the web and phone.

Design:

consultancy hero section

Link to full design:

```
https://xd.adobe.com/view/b98caf27-6549-49e5-8ed8-c5234788df61-ecdf/screen/cc35e19a-24f2-42ce-8803-ee2942839528/
```
 
### Technical Analysis

This feature can be found in libs/features/pages/consultancy-page inside a components folder. The component's name is consultancy-hero-section.

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the design above.
2. The component is clean and does not have more than 30 lines
3. The component SCSS uses CSS variables for colour, font, ... Connected to the global style.scss
5. The design is responsive for screen, tablet and phone

### PR DESCRIPTION
**Pull request by Yhworld**
# Description

A Hero section on the consultancy page that captures attention and introduces the page content.

Fixes #176 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Screenshot (optional)

ghd

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Yhworld** [comment]
@LemmyMwaura

File: libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.scss [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    width: 100vw;
+    overflow: hidden; /* Hides the overflowing content */
+    position: relative;
+}
+
+.hero-section{
+    position: absolute;
+    z-index: 5999999999; 
+    margin-top: 5rem;
+    // margin-left: 20rem;
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.scss:13]: To eliminate dead code, commented out code should be removed.

File: libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+})
+export class ConsultancyHeroSectionComponent {
+  heroTexts: HeroText = {
+    title:"Learning Design",
+    description:"Your organisation has the potential to turn every task into an opportunity to learn. From growing crops, to nursing someone to good health, to developing the next-generation technology. Let’s work together to make the magic happen."
+  }
+
+  buttonData: SliderButtonData = {
+    text: "Unlock your potential"
+  }
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.ts:19]: To keep the component lean and ensure readability and maintainability, the 'heroTexts' and 'buttonData' variable content can be moved to a Typescript file in the data module.

**Review by AnnaL001 — CHANGES_REQUESTED** [review]
Nicely done, just a few adjustments.

Also, the left margin of the hero section and the top bar can be reduced, to match the specified design.

**Comment by Yhworld** [comment]
@AnnaL001  done kindly review

**Review by LemmyMwaura — APPROVED** [review]

**Review by LemmyMwaura — APPROVED** [review]

**Pull request merged**